CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [#5](https://github.com/dblock/iex-ruby-client/pull/5): Added `IEX::Resources::OHLC#get` - [@jromanovs](https://github.com/jromanovs). 
 * [#5](https://github.com/dblock/iex-ruby-client/pull/5): Added `IEX::Resources::OHLC#market` - [@jromanovs](https://github.com/jromanovs). 
 * [#9](https://github.com/dblock/iex-ruby-client/pull/9): Added `IEX::Errors::ClientError` - [@rodolfobandeira](https://github.com/rodolfobandeira).
+* [#10](https://github.com/dblock/iex-ruby-client/pull/10): Added `IEX::Resources::Logo#get` - [@rodolfobandeira](https://github.com/rodolfobandeira).
```

Comment:
I missed this, this changelog entry and the one above belongs in 0.3.4 above, not in the released 0.3.3.

Reply:
Fixed. Please let me know if that is in the correct format. I guess you'll add the new entry `0.3.5` and edit the date when you release the `0.3.4` right?

Thanks again!

Reply:
Right.